test/com/serotonin/mango/rt/dataSource/meta/MetaPointLocatorRTExecutionDelayTimeoutRunTest.java

```diff
@@ -0,0 +1,92 @@
+package com.serotonin.mango.rt.dataSource.meta;
+
+import com.serotonin.mango.Common;
+import com.serotonin.mango.DataTypes;
+import com.serotonin.mango.rt.RuntimeManager;
+import com.serotonin.mango.rt.dataImage.DataPointRT;
+import com.serotonin.mango.rt.dataImage.PointValueTime;
+import com.serotonin.mango.vo.dataSource.meta.MetaPointLocatorVO;
+import com.serotonin.timer.RealTimeTimer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+import utils.mock.MockUtils;
+
+import java.util.ArrayList;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+
+@RunWith(PowerMockRunner.class)
+@PowerMockRunnerDelegate(Parameterized.class)
+@PrepareForTest({MetaPointLocatorRT.ScheduledUpdateTimeout.class, MetaPointLocatorRT.class})
+@PowerMockIgnore({"com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*", "org.w3c.*", "com.sun.org.apache.xalan.*",
+        "javax.activation.*", "javax.management.*"})
+public class MetaPointLocatorRTExecutionDelayTimeoutRunTest {
+
+    @Parameterized.Parameters(name= "{index}: Update event type: {0}, Expected times of invocation: {1}")
+    public static Object[] data() {
+        return new Object[] {
+                new Object[] {MetaPointLocatorVO.UPDATE_EVENT_CRON, 1},
+                new Object[] {Common.TimePeriods.MINUTES, 1},
+                new Object[] {Common.TimePeriods.HOURS, 1},
+                new Object[] {Common.TimePeriods.DAYS, 1},
+                new Object[] {Common.TimePeriods.WEEKS, 1},
+                new Object[] {Common.TimePeriods.MONTHS, 1},
+                new Object[] {Common.TimePeriods.YEARS, 1},
+        };
+    }
+
+    private final DataPointRT dataPoint = mock(DataPointRT.class);
+    private final MetaDataSourceRT dataSource = mock(MetaDataSourceRT.class);
+    private final int expectedInvocationTimes;
+    MetaPointLocatorRT metaPointLocatorRT;
```

Comment:
What is this?
Why are the fields above initialized correctly and these are not?